bower.json

```diff
@@ -20,7 +20,8 @@
     "iron-flex-layout": "^1.0.0",
     "cosmoz-i18next": "neovici/cosmoz-i18next#master",
     "cosmoz-dialog": "neovici/cosmoz-dialog#master",
-    "iron-list": "PolymerElements/iron-list#^2.0.4"
+    "iron-list": "PolymerElements/iron-list#^2.0.4",
+    "cosmoz-treenode": "Neovici/cosmoz-treenode"
```

Comment:
@nomego  Ok, not sure I get this and I think its related with the behavior I created. 

Do I understand correctly, `cosmoz-treenode-navigator` gets a tree from outside the element? If so, how can I be sure that certain methods of `Cosmoz.DefaultTree` are implemented in the tree data the navigator gets from outside in order to work with them inside of the navigator? Do we define, the data (tree object) we get from outside needs do be a `Cosmoz.DefaultTree` (or derived from) object?

My understanding was, we get "node-like" data from outside and use the Cosmoz.Tree or DefaultTree methods inside the navigator. => wrong?